frontend/src/components/my_calendar/MyCalendar/MyCalendar.tsx

```diff
@@ -28,15 +33,36 @@ const MyCalendar = (props: MyCalendarProps) => {
     today,
     handlers: { handlePrevButtonClick, handleNextButtonClick },
   } = useCalendar();
-  const schedules = useFetchMySchedules(year, month);
+
   const { teamPlaces } = useTeamPlace();
+
+  const { firstDateOfCalendar, lastDateOfCalendar } =
+    getFirstLastDateOfCalendar(year, month);
+
+  const firstDateOfPreviousMonthCalendar = new Date(
```

Comment:
이전달과 다음달도 위의 친구처럼 할 수 있을 것같은데 따로 계산을 해야하는 이유가 뭐야?

```suggestion
 const { firstDateOfPreviousMonthCalendar, lastDateOfPreviousMonthCalendar } =
    getFirstLastDateOfCalendar(year, month -1);
```